krux/io.py

```diff
@@ -82,16 +82,21 @@ def is_regex(thing):
 
         log.debug('Applying output filters: %s' % [r.pattern for r in filters])
 
-        try:
-            ### If you already gave us a list, great. Otherwise, split on whitespace.
-            command = command if isinstance(command, (list, tuple)) else command.split()
+        # use shell param if set, otherwise use shell if we were passed a string
+        # i.e. let the shell parse the string rather than just splitting it
```

Comment:
This comment tells us *what* you are doing, but not *why*. Can you elaborate more on the use case for this change, please, so we understand it later when we are looking at this code?